kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -67,6 +67,9 @@
     'apiserver_storage_list_fetched_objects_total': 'storage_list_fetched_objects_total',
     'apiserver_storage_list_evaluated_objects_total': 'storage_list_evaluated_objects_total',
     'apiserver_storage_list_returned_objects_total': 'storage_list_returned_objects_total',
+    # For Kubernetes >= 1.26
+    # https://github.com/kubernetes/kubernetes/pull/112690
+    'kubernetes_feature_enabled': 'kubernetes_feature_enabled',
```

Comment:
I'm wondering it you should remap to `feature_enabled`. the result will be `kube_apiserver.feature_enabled`.

If we only get the kube_apiserver feature, we could remove the `kubernetes_` prefix from the metric name

Reply:
after discussion with @tbavelier, he confirms that all components features will be listed from the api-server metrics endpoints